tests/formatter_tests.ts

```diff
@@ -556,7 +550,7 @@ function multiStaves(options: TestOptions): void {
   ];
 
   const staveYs = [20, 130, 250];
-  let staveWidth = width + getGlyphWidth('gClef') + getGlyphWidth('timeSig8') + Stave.defaultPadding;
+  let staveWidth = width + getGlyphWidth('E050' /*gClef*/) + getGlyphWidth('E088' /*timeSig8*/) + Stave.defaultPadding;
```

Comment:
Since we aren't using the glyph name anymore.... we probably don't need to do the parseInt() and String.fromCharCode() above. Why not just pass in the actual unicode string directly to el.setText()?

Reply:
reworked